### PR TITLE
1.3

### PR DESCRIPTION
**Pull request by zoemccormick**
**Please provide enough information so that others can review your pull request:**

1. Explain the **details** for making this change. What existing problem does the pull request solve? If it resolves an existing issue, be sure to use a Github [keyword](https://help.github.com/en/articles/closing-issues-using-keywords) to automatically close it.

<br/><br/>

2. What **changes to custom.yaml** are required?

<br/><br/>

3. Have you documented any additional setup steps or configurations options?

<br/><br/>

4. Demonstrate the code is solid. Example: The exact commands you ran and their output, screenshots / videos if the pull request changes UI.

<br/><br/>

### CODE REVIEW

File: fabric/Chart.yaml [hunk 1]
```diff
@@ -15,15 +15,15 @@ keywords:
 dependencies:
   - name: control-api
     repository: file://./control-api
-    version: '3.0.2'
+    version: '3.0.3'
 
   - name: control
     repository: file://./control
-    version: '2.2.5'
+    version: '2.2.6'
```
Comment (chrisbsmith) [inline fabric/Chart.yaml:22]: we need a consistent versioning scheme. If we're bumping control-api to 3.X we should do the same for control and jwt

File: fabric/Chart.yaml [hunk 2]
```diff
@@ -15,15 +15,15 @@ keywords:
...
 
   - name: jwt
     repository: file://./jwt
-    version: '2.2.3'
+    version: '2.2.4'
```
Comment (chrisbsmith) [inline fabric/Chart.yaml:26]: same version bump comment here

File: fabric/control/Chart.yaml [hunk 3]
```diff
@@ -3,7 +3,7 @@ appVersion: 1.1.0
 description: Deploys the Grey Matter 2.0 control server
 name: control
 home: https://greymatter.io
-version: 2.2.5
+version: 2.2.6
```
Comment (chrisbsmith) [inline fabric/control/Chart.yaml:6]: version bump

File: fabric/jwt-gov/Chart.yaml [hunk 4]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.0.0
 description: A Helm chart to deploy Grey Matter JWT Security Gov
 name: jwt-gov
-version: 2.2.3
+version: 2.2.4
```
Comment (chrisbsmith) [inline fabric/jwt-gov/Chart.yaml:5]: version bump

File: fabric/jwt/Chart.yaml [hunk 5]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.0.0
 description: A Helm chart to deploy Grey Matter JWT Security
 name: jwt
-version: 2.2.3
+version: 2.2.4
```
Comment (chrisbsmith) [inline fabric/jwt/Chart.yaml:5]: version bump

File: sense/Chart.yaml [hunk 6]
```diff
@@ -15,12 +15,12 @@ keywords:
 dependencies:
   - name: catalog
     repository: file://./catalog
-    version: '3.0.1'
+    version: '3.0.2'
 
   - name: dashboard
     repository: file://./dashboard
-    version: '2.2.4'
+    version: '2.2.5'
```
Comment (chrisbsmith) [inline sense/Chart.yaml:22]: version bump

File: sense/Chart.yaml [hunk 7]
```diff
@@ -15,12 +15,12 @@ keywords:
...
 
   - name: slo
     repository: file://./slo
-    version: '2.2.5'
+    version: '2.2.6'
```
Comment (chrisbsmith) [inline sense/Chart.yaml:26]: version bump

File: sense/dashboard/Chart.yaml [hunk 8]
```diff
@@ -1,8 +1,8 @@
 apiVersion: v2
-appVersion: 3.4.2
+appVersion: 4.0.0
 description: A Helm chart to deploy Grey Matter Dashboard
 name: dashboard
-version: 2.2.4
+version: 2.2.5
```
Comment (chrisbsmith) [inline sense/dashboard/Chart.yaml:5]: version bump

File: sense/slo/Chart.yaml [hunk 9]
```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 appVersion: 1.1.5
 description: A Helm chart to deploy Grey Matter Service Level Objectives
 name: slo
-version: 2.2.5
+version: 2.2.6
```
Comment (chrisbsmith) [inline sense/slo/Chart.yaml:5]: version bump

**Review by chrisbsmith — CHANGES_REQUESTED** [review]
Overall, I'm very good with this, but if we need to upgrade all of chart versions as well.

**Review by kaitmore — APPROVED** [review]
LGTM

**Review by chrisbsmith — APPROVED** [review]

**Pull request merged**